core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/upgrade/ClusterUpgradeFailedRequest.java

```diff
@@ -8,7 +8,7 @@
 import com.sequenceiq.cloudbreak.cloud.model.catalog.Image;
 import com.sequenceiq.cloudbreak.reactor.api.event.StackEvent;
 
-public class ClusterUpgradeFailedCmSyncRequest extends StackEvent {
+public class ClusterUpgradeFailedRequest extends StackEvent {
```

Comment:
this rename might kill restarting flows

Reply:
I believe it will retrigger from the `ClusterUpgradeActions.doExecute` method, thus it wont fail the restarted flows.

Reply:
I have double-checked, flowlog (and flowchainlog) table doesnt contain any information about the event handlers, it stores only the flow steps, thus it will retrigger the latest flow step (*Action).